test/linux/aliastool.c

```diff
@@ -368,7 +368,7 @@ void eepromtool(char *ifname, int slave, int mode, char *fname)
             if (mode == MODE_WRITEALIAS)
 			{
 			  if(eeprom_writealias(slave, alias))
-			    printf("Alias %4.4X written successfully to slave %d\n");
+			    printf("Alias %4.4X written successfully to slave %d\n", (unsigned)alias, slave);
```

Comment:
Is the explicit cast necessary? There are other similar calls for fprintf not having it.

Reply:
No, the explicit cast is not necessary.  
I checked the documentation, and since the argument undergoes [integer promotion](https://en.cppreference.com/w/c/language/conversion#Default_argument_promotions), it is not needed.